nodepool/scripts/convert_node_to_xenserver.sh

```diff
@@ -692,6 +692,9 @@ function configure_hostname {
 
     if [ -n "$HOSTNAME" ]; then
         {
+            # Update hosts to avoid warning when running sudo command:
+            # e.g. sudo: unable to resolve host dsvm-devstack-rax-iad-nodepool-726969
+            echo "sudo sed -i 's/^127.0.1.1.*$/127.0.1.1 '$HOSTNAME'/g' /etc/hosts"
```

Comment:
That's in the existing script: https://github.com/citrix-openstack/os-ext-testing/blob/master/nodepool/scripts/prepare_node.sh#L33

And it seems that's recommended by debian to use different entry for localhost and <hostname> ; that's a workaround for some software bugs:
https://www.debian.org/doc/manuals/debian-reference/ch05.en.html#_the_hostname_resolution